### PR TITLE
DevWs for NIE-TECH CO., LTD. containing containing Z-Wave Dual Switch

### PR DESCRIPTION
**Pull request by sky-nie**

### CODE REVIEW

File: devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy [hunk 1]
```diff
@@ -28,6 +28,7 @@ metadata {
 		fingerprint mfr: "0000", cc: "0x5E,0x25,0x27,0x81,0x71,0x60,0x8E,0x2C,0x2B,0x70,0x86,0x72,0x73,0x85,0x59,0x98,0x7A,0x5A", ccOut: "0x82", ui: "0x8700", deviceJoinName: "Aeotec Dual Nano Switch 1"
 		fingerprint mfr: "0258", prod: "0003", model: "008B", deviceJoinName: "NEO Coolcam Light Switch 1"
 		fingerprint mfr: "0258", prod: "0003", model: "108B", deviceJoinName: "NEO coolcam Light Switch 1"
+        fingerprint mfr: "0312", prod: "C000", model: "C004", deviceJoinName: "EVA LOGIK Smart Plug 2CH 1"
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy]: You probably should fix indentation here.

Comment (MAblewicz) [reply devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy]: @sky-nie    Use tabs instead of spaces.

**Comment by sky-nie** [comment]
I've fixed the fingerprint and committed the change back to this branch, but let me know if I should have handled that a different way.

**Comment by shrakuma** [comment]
@MAblewiczS @greens @tpmanley  if all changes are good. can you please merge this

**Review by greens — APPROVED** [review]

**Pull request merged**